.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: node_js
 node_js:
+  - "8"
   - "7"
+  - "6"
 cache: yarn
+before_install: yarn global add greenkeeper-lockfile@1
+before_script: greenkeeper-lockfile-update
 script:
   - yarn lint
   - yarn test -- --coverage
-branches:
-  only:
-    - master
-notifications:
-  email: false
-
```

Comment:
I removed this, is that okay? I think travis should run on all branches and i definitely think that getting notified about failures is a good thing.

But can also revert this

Reply:
Go ahead :)